mitodl/bootcamp-ecommerce
# Fixed resume form to update correctly after upload

#### What are the relevant tickets?
Closes #728 

#### What's this PR do?
- Updates the resume/linkedin form to show the resume filename after it's successfully uploaded
- Adds validation to the form so that either a resume or a linkedin URL must be provided, and a non-URL in the linkedin input will also fail

#### How should this be manually tested?
All of these require you to click the resume/LinkedIn URL link in a detail section of the application dashboard (which will open a drawer)

- Upload a resume by dropping a file or using the file browser. You should see a caption under the dropzone box that shows the filename of the resume you just uploaded
- You should see a validation message in these scenarios: 
  - Click submit with neither a resume nor a linkedin URL
  - Click submit with some text in the linkedin input that is not a valid URL
- Clicking submit should close the drawer if the linkedin URL is valid and/or a resume has been successfully uploaded

#### Any background context you want to provide?
The mobile screen looks off b/c our styling of drawers has been very inconsistent. I'll figure this out with Abdul

#### Screenshots (if appropriate)
<img width="1239" alt="ss 2020-06-23 at 12 37 18 " src="https://user-images.githubusercontent.com/14932219/85433596-a77aa580-b552-11ea-8ea8-afce315efa9f.png">
<img width="417" alt="ss 2020-06-23 at 12 38 22 " src="https://user-images.githubusercontent.com/14932219/85433612-ab0e2c80-b552-11ea-93bb-0e3d30fd5719.png">
<img width="543" alt="ss 2020-06-23 at 12 40 29 " src="https://user-images.githubusercontent.com/14932219/85433613-ab0e2c80-b552-11ea-9c86-dee5659cd105.png">
<img width="543" alt="ss 2020-06-23 at 12 40 35 " src="https://user-images.githubusercontent.com/14932219/85433614-ab0e2c80-b552-11ea-9b43-526480496559.png">
<img width="1226" alt="ss 2020-06-23 at 12 40 48 " src="https://user-images.githubusercontent.com/14932219/85433615-aba6c300-b552-11ea-958e-f5a83a71bd8b.png">
